build.gradle

```diff
@@ -45,6 +66,21 @@ nohttp {
 allprojects {
 	group = 'org.springframework.amqp'
 
+	apply plugin: "io.spring.dependency-management"
+
+	dependencyManagement {
+		resolutionStrategy {
+			cacheChangingModulesFor 0, 'seconds'
+		}
+		applyMavenExclusions = false
+		generatedPomCustomization {
+			enabled = false
+		}
+		imports {
+			mavenBom "com.fasterxml.jackson:jackson-bom:$jackson2Version"
```

Comment:
I wouldn't do this in all the modules of the project: only in those where we need such a dependency.
What makes you to move all the versions stuff to the top level?

Reply:
1. ok
2. because the plugin can't resolve the bom version - the Spring PR did the same.